hue_lo: 0
hue_hi: 123.0
sat_lo: 71.4
sat_hi: 255.0
val_lo: 61.09
val_hi: 255.0
lines_min_len: 10
lines_angle_lo: 0
lines_angle_hi: 360
